backends/apple/coreml/test/test_coreml_partitioner.py

```diff
@@ -71,23 +71,15 @@ def test_vit_skip_conv(self):
             )
         )
 
-        conv_block = ["aten.convolution.default", "executorch_call_delegate"]
```

Comment:
What does it mean?

Reply:
In this test case (vit model), vit is fully coreml resident, so we expect the partitioned graph to simply be
```
"aten.convolution.default" -> "executorch_call_delegate" -> "getitem"
```
However, https://github.com/pytorch/pytorch/pull/131863 introduced safe softmax, which involves `torch.any` op that was not supported in coremltools 8.0, so the partitioned graph had 12 blocks separated by `torch.any`

As coremltools 8.1 added `torch.any` support, we get back to the simple no-break graph